hack/kiali/func-sm.sh

```
@@ -0,0 +1,474 @@
+  #infomsg "done."
+
+  # "latest" is not a supported version when using a released version of Sail operator.
+  # We try to determine the latest version of Istio supported by examining the CRD.
+  if [ "${istio_version}" == "latest" ]; then
+    istio_version="$(${OC} get crd istios.sailoperator.io -o json | jq -r '
+      (.spec.versions | map(select(.storage == true)) | first) as $st
+      | (if ($st | type) == "object" then $st
+         else (.spec.versions | map(select(.served == true and (.name | test("^v[0-9]+$")))) | sort_by(.name) | last)
+         end)
+      | .schema.openAPIV3Schema.properties.spec.properties.version.default // empty')"
+    if [ -z "${istio_version}" -o "${istio_version}" == "null" ]; then
```

Shellcheck (0.11.0)

[warning] Prefer [ p ] || [ q ] as [ p -o q ] is not well defined.

(SC2166)